push_notifications/services/exceptions.py

```diff
@@ -6,3 +6,7 @@
 
 class InvalidPushNotificationError(ImproperlyConfigured):
     pass
+
+
+class NotificationError(Exception):
```

Comment:
I think `InvalidPushNotificationError` is extending `ImproperlyConfigured` which will be raised during intialization of any service. Whereas `NotificationError` is generic exception which should be raised when sending notifications/ getting responses etc.